src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -209,7 +209,7 @@ function tryRemoveElement(elem) {
         /**
          * @type {any | null | undefined}
          */
-        #currentSubtitlesOctopus;
+        #currentJASSUB;
```

Comment:
I think it would be better to use generic variable names where possible to avoid needing to change this again at some point. Maybe something like `currentAssRenderer`?

Reply:
I doubt any1's batshit insane enough to make another ASS renderer, speaking from experience of how much sanity I've lost over trying to implement mine, so I don't think there's actually a need?

Reply:
no followup, so i'm gonna assume its not needed

Reply:
You wouldn't think this would happen often, but this would be the third ass/ssa rendering library we've used in 4 years.

Reply:
lets hope it's the last you'll need lol, renamed it @thornbill